openedx_cas/backends.py

```diff
@@ -2,9 +2,63 @@
 implementation.
 """
 from social_core.backends.base import BaseAuth
+from social_core.exceptions import AuthFailed
+from django.conf import settings
+from django_cas_ng.backends import CASBackend
+import urllib.parse
+from django_cas_ng.utils import get_cas_client
+from django.contrib.auth import get_user_model
 
+import logging
 
-class CASAuth(BaseAuth):
+UserModel = get_user_model()
+
+logger = logging.getLogger(__name__)
+
+
+class CASBackendOverride(CASBackend):
+
+    def cas_validation(self, request, ticket, service):
+        client = get_cas_client(service_url=service, request=request)
+        username, attributes, pgtiou = client.verify_ticket(ticket)
+
+        if not username:
+            message = f"Couldn't find username associated with ticket{ticket}"
```

Comment:
Isn't a data leakage to log the ticket?

Reply:
@mariajgrimaldi It seems the ticket expiration can be modified, if it's a high value (some minutes) it can dangerous.

Let's remove it for now.